README.md

```diff
@@ -26,8 +26,13 @@ Please checkout [ML on GCP](https://github.com/GoogleCloudPlatform/ml-on-gcp) gu
 * [Reddit](reddit_tft) - Predict the score of a Reddit thread using a wide and deep model
 * [Cifar10](https://github.com/tensorflow/models/tree/master/tutorials/image/cifar10_estimator) - Classify image content (training on multiple GPUs)
 
-### TPU (Contrib)
-* [Resnet TPU](tpu)
+### TPU
+* [MNIST on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/mnist)
+* [ResNet-50 on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/resnet)
+* [Inception on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/inception)
+* [Inception V3 on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/inception-v3-advanced)
+* [RetinaNet on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/retinanet)
+* [Transformed - Attention Model on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/transformer)
```

Comment:
I'm not sure if "transformed-attention" is a well known term. How about this instead:

```
[Transformer model on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/transformer) for natural-language translation, language modeling, and sentiment classification
```

Reply:
done

Reply:
Details captured with T2T